packages/builder/src/util.ts

```diff
@@ -4,21 +4,38 @@ import { Buffer } from 'buffer';
 import _ from 'lodash';
 import { JsonFragment } from '@ethersproject/abi';
 
+import { z } from 'zod';
+
 import { ChainDefinition } from '.';
 import { ChainDefinitionProblems } from './definition';
 import { ChainBuilderContext, ChainArtifacts } from './types';
 
 import { CannonWrapperGenericProvider } from './error/provider';
 
-export const ChainDefinitionScriptSchema = {
-  properties: {
-    exec: { type: 'string' },
-  },
-  optionalProperties: {
-    args: { elements: { type: 'string' } },
-    env: { elements: { type: 'string' } },
-  },
-} as const;
+// FIXME: Why not move this to keepers.ts since this isnt shared by any other file?
```

Comment:
This used to be used be used for both keepers and `run` step (which was moved into cli obviously)

So I would agree it makes sense to make this change